.github/workflows/USERSERVICE_AUTO_RUN.yml

```
@@ -0,0 +1,58 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        java-version: [11]  # Specify the JDK version(s) here
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3  # Checks out your repository
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)